### ISSUE TITLE
Migration from Jbehave to Cucumber

### ISSUE THREAD

**Issue by vaibhav2701** [serenity-bdd/serenity-core#1380]
Hi,

We have been using a serenity-jbehave framework for more than 2 years now for our functional test cases. But it seems Cucumber is now more popular and we decided to switch to serenity-cucumber setup.

The challenge is in our existing steps we have something like what is shown below at multiple places and it seems Cucumber is not happy about it. It wants unique steps for each step. Is there some workaround for this?? We can't afford to make so many changes at so many places.
 
'@Given ( "user is logged into the application" )
@When ( "user is logged into the application" )
@Then ( "user is logged into the application" )
    public void givenUserIsLoggedIntoTheApplication() {
        System.out.println( "Login is completed" );
    }'

**Comment by mleegwt** [comment]
This looks like a Cucumber issue https://github.com/cucumber/cucumber-jvm/issues/1341

I don’t think this is a serenity(-cucumber) issue.

**Comment by wakaleo** [comment]
Use any one annotation, Cucumber will match any of the keywords.

**Comment by vaibhav2701** [comment]
Ok so I guess I did not state the complete problem. Apart from @Given, @When and @Then pointing to same method the other issue which we have is @Alias tag. We have used it generously in our JBehave framework but Cucumber it seems does not support it. The language of the steps is different but they refer to same implementation method. I tried looking for an answer in Cucumber groups as well but could not locate one.
